### PR TITLE
Add serialization customization section

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -309,6 +310,10 @@ namespace Azure.Service.Models
 
 </details>
 
+### Customize serialization/deserialization methods
+
+Use the [Replace any generated member](#replace-any-generated-member) approach to replace Serialize/Deserialize method with a custom implementation.
```
Comment (annelo-msft) [inline readme.md:315]: Would it make sense to link to this sample? https://github.com/Azure/autorest.csharp/blob/feature/v3/test/TestProjects/TypeSchemaMapping/Customized/CustomizedModel.cs#L20

Comment (pakrym) [reply readme.md:315]: Want to keep this document self sufficient, I tend to overcomplicate test classes by adding to many features to them. Added a sample.

File: readme.md [hunk 2]
```diff
@@ -314,6 +314,82 @@ namespace Azure.Service.Models
...
+                  if (property.Value.ValueKind == JsonValueKind.Null)
+                  {
+                      continue;
+                  }
+                  name = property.Value.GetString();
+                  continue;
+              }
+          }
+          // WORKAROUND: server never sends color, default to black
+          colod = "black";
```
Comment (annelo-msft) [inline readme.md:380]: nit: compiler error here.

**Comment by maririos** [comment]
Thank you both for adding this :)

**Pull request merged**